liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -27,6 +27,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.io.FileWriter;
```

Comment:
Add 2020 to the copyright at the top.

Reply:
Not needed anymore since we are not making changes to this file